### PR TITLE
Updated package version

### PR DESCRIPTION
**Pull request by vladvildanov**
Updated package version to represent current version

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -2,5 +2,5 @@ package redis
 
 // Version is the current release version.
 func Version() string {
-	return "9.6.1"
+	return "9.7.0-beta.1"
```
Comment (ofekshenawa) [inline version.go:5]: Typo? 
Also, we have already released a beta version in this format: 9.7.0-b1, so maybe we want to stick to this format

**Review by ofekshenawa — CHANGES_REQUESTED** [review]
Please note that the version needs to be updated in additional places. Make sure to check all relevant files and documentation where the version is referenced to ensure consistency.
f.e https://github.com/redis/go-redis/pull/2917/files

File: version.go (hunk 1 above)
Comment (vladvildanov) [reply version.go:5]: I guess we have to stick to the conventions in go community

https://go.dev/doc/modules/version-numbers

Comment (vladvildanov) [reply version.go:5]: Previously the format was broken, f.e 9.6.0b1

**Comment by vladvildanov** [comment]
@ofekshenawa But if we update it in submodules they will point to a version that doesn't exists yet right? My IDE is complaining when I'm doing so

**Pull request merged**